_includes/releases/v23.1/v23.1.0-alpha.5.md

```diff
@@ -0,0 +1,166 @@
+  - `sql.mem.internal.session.prepared.current`
+  - `sql.mem.internal.session.prepared.max-avg`
+  - `sql.mem.internal.session.prepared.max-count`
+  - `sql.mem.internal.session.prepared.max-max`
+  - `sql.mem.internal.session.prepared.max-p50`
+  - `sql.mem.internal.session.prepared.max-p75`
+  - `sql.mem.internal.session.prepared.max-p90`
+  - `sql.mem.internal.session.prepared.max-p99`
+  - `sql.mem.internal.session.prepared.max-p99.9`
+  - `sql.mem.internal.session.prepared.max-p99.99`
+  - `sql.mem.internal.session.prepared.max-p99.999`
+  - `sql.mem.sql.session.prepared.current`
+  - `sql.mem.sql.session.prepared.max-avg`
+  - `sql.mem.sql.session.prepared.max-count`
+  - `sql.mem.sql.session.prepared.max-max`
+  - `sql.mem.sql.session.prepared.max-p50`
+  - `sql.mem.sql.session.prepared.max-p75`
+  - `sql.mem.sql.session.prepared.max-p90`
+  - `sql.mem.sql.session.prepared.max-p99`
+  - `sql.mem.sql.session.prepared.max-p99.9`
+  - `sql.mem.sql.session.prepared.max-p99.99`
+  - `sql.mem.sql.session.prepared.max-p99.999`
+
+    [#97590][#97590]
+
+- Active execution information is now shown even when there is a max size limit error. [#97662][#97662]
+- "Retrying" is no longer a status shown in the Jobs page. [#97505][#97505]
+
+<h3 id="v23-1-0-alpha-5-bug-fixes">Bug fixes</h3>
+
+- Transaction uncertainty intervals are now correctly configured for reverse scans, to prevent reverse scans from serving stale reads when clocks in a cluster are skewed. [#97443][#97443]
+- The formatting of uniqueness violation errors now matches the corresponding errors from PostgreSQL. [#96914][#96914]
+- Fixed a bug in `ALTER TABLE ... ADD COLUMN` when the new column name requires quoting due to mixed case or special characters and the statement is not run in an explicit or multi-statement transaction. [#97514][#97514]
+- Fixed a bug when formatting `CREATE` statements for user-defined types which require quoting which might prevent those statements from round-tripping. [#97514][#97514]
+- Using subqueries in UDFs without an `AS` clause is now supported, for consistency with the syntax supported outside of UDFs. [#97515][#97515]
+- Fixed a rare bug existing since before v22.1.x that could cause a projected expression to replace column references with the wrong values. [#97554][#97554]
+- Cross-descriptor validation on lease renewal is now disabled, because it can starve schema changes when there are many descriptors with many foreign key references [#97630][#97630]
+- Fixed a bug with pagination on the Schema Insights page. [#97640][#97640]
+- Columns referenced in partial index predicates and partial unique constraint predicates can no longer be dropped. The `ALTER TABLE .. DROP COLUMN` statement now returns an error with a suggestion to drop the indexes and constraints first. This is a temporary safe-guard to prevent users from hitting [#96924][#96924]. This restriction will be lifted when that bug is fixed. [#97372][#97372]
```

Comment:
Link to the SQL statement doc.

```suggestion
- Columns referenced in partial index predicates and partial unique constraint predicates can no longer be dropped. The `ALTER TABLE .. DROP COLUMN` statement now returns an error with a suggestion to drop the indexes and constraints first. This is a temporary safeguard to prevent users from hitting [#96924][#96924]. This restriction will be lifted when that bug is fixed. [#97372][#97372]
```

Reply:
I don't think this is about the SQL Statement page.

Reply:
I meant the doc for the SQL statement: `alter-table.html#drop-column`